libs/components/lookup/testing/src/country-field/country-field-search-result-harness.ts

```diff
@@ -0,0 +1,34 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+
+import { SkyAutocompleteSearchResultHarness } from '../autocomplete/autocomplete-search-result-harness';
+
+import { SkyCountryFieldSearchResultHarnessFilters } from './country-field-search-result-harness-filters';
+
+/**
+ * Harness for interacting with an autocomplete search result in tests.
+ */
+export class SkyCountryFieldSearchResultHarness extends SkyAutocompleteSearchResultHarness {
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyCountryFieldSearchResultHarness` that meets certain criteria.
+   */
+  public static override with(
+    filters: SkyCountryFieldSearchResultHarnessFilters,
+  ): HarnessPredicate<SkyCountryFieldSearchResultHarness> {
+    return new HarnessPredicate(
+      SkyCountryFieldSearchResultHarness,
+      filters,
+    ).addOption('textContent', filters.text, async (harness, text) =>
+      HarnessPredicate.stringMatches(await harness.getText(), text),
+    );
+  }
+
+  /**
+   * Returns the value of the search result's descriptor property.
+   * This is not needed by country field because it is always set to the country name.
+   * @internal
+   */
+  public override async getDescriptorValue(): Promise<string> {
```

Comment:
can we be more descriptive that this is being marked @internal to remove this value from the JSdocs for country field search results?

Reply:
Updated